Code-publishing.md

```diff
@@ -0,0 +1,9 @@
+Code Publishing
+---------------
+
+Any code you write may be useful for others, could facilitate collaboration and could potentially be cited in publications. Publishing your code facilitates these outcomes.
+
+### Skill Set #1
+* Software publication workflow
```

Comment:
What workflow are you talking about?

Reply:
But then I guess it should read like this:
### Skill Set #1: Software publication workflow
 * ...
 * Publication with Zenodo
Because I would say the publication process is part of the workflow?

Reply:
maybe something like this is clearer?:
skill set 1: publishing software
1) how to publish/what publishing means ... or something similar 
2) sharing code on github
3) publishing with Zenodo

I'm still getting my head around what we're trying to achieve :-)